### ISSUE TITLE
[k8s] Fail to ssh into the head node on k8s

### ISSUE THREAD

**Issue by alex000kim** [skypilot-org/skypilot#4461]
The cluster launches and executes the task sucessfully:

```
sky launch -c test --cloud kubernetes -- echo hi
```
I fail to ssh into it:

```
$ ssh test
Bad stdio forwarding specification '2600:1f16:cf3:2521:cf7c::8:22'
Connection closed by UNKNOWN port 65535
```

**Comment by romilbhardwaj** [comment]
Thanks for the report @alex000kim! Looks like this is related to running on a k8s cluster with ipv6. Two steps for us:

1. Replicate on our end. 
2. Fix our proxycommand script to have socat/kubectl port-forward work with ipv6.

**Comment by romilbhardwaj** [comment]
Closed with #4497.
